scripts/lisk_snapshot.sh

```diff
@@ -70,93 +81,132 @@ parse_option() {
         elif [ "$OPTARG" == "highest" ]; then
           SNAPSHOT_ROUND="$OPTARG"
         else
-          echo "Snapshot flag must be a greater than 0 or set to highest"
+          echo "$( date +'%Y-%m-%d %H:%M:%S' ) Snapshot flag must be a greater than 0 or set to highest"
           exit 1
         fi ;;
 
       g) GENERIC_COPY="Y" ;;
 
+      v) PGSQL_VACUUM="Y" ;;
+
+      m)
+        if [ "$OPTARG" -ge 1 ]; then
+          PGSQL_VACUUM_DELAY=$OPTARG
+        else
+          echo "$( date +'%Y-%m-%d %H:%M:%S' ) Invalid number for vacuum delay in minute(s)."
+          exit 1
+        fi ;;
+
       ?) usage; exit 1 ;;
 
-      :) echo 'Missing option argument for -'"$OPTARG" >&2; exit 1 ;;
+      :) echo "$( date +'%Y-%m-%d %H:%M:%S' ) Missing option argument for -$OPTARG" >&2; exit 1 ;;
+
+      *) echo "$( date +'%Y-%m-%d %H:%M:%S' ) Unimplemented option: -$OPTARG" >&2; exit 1 ;;
 
-      *) echo 'Unimplemented option: -'"$OPTARG" >&2; exit 1 ;;
     esac
   done
 }
 
 usage() {
-  echo "Usage: $0 [-t <snapshot.json>] [-s <config.json>] [-b <backup directory>] [-d <days to keep>] [-r <round>] [-g]"
+  echo -e "\nUsage: $0 [-t <snapshot.json>] [-s <config.json>] [-b <backup directory>] [-d <days to keep>] [-r <round>] [-g] [-v] [-m <vacuum delay>]\n"
```

Comment:
Remove this flag and just set vaccuum to always on using a hardcoded cycle of 15-30 minutes or so.